min_compatible_versions.csv

```diff
@@ -0,0 +1,188 @@
+This file was generated by scripts/min_compatible_versions.py
+pkg_name,min_version
+Flask-Cache,~=0.13.1
+Jinja2,~=2.11.0
+SQLAlchemy,==2.0.22
+WebTest,0
+Werkzeug,<1.0
+ai21,0
+aiobotocore,~=1.4.2
+aiofiles,0
+aiohttp,~=3.7
+aiohttp_jinja2,~=1.5.0
+aiomysql,~=0.1.0
+aiopg,~=0.16.0
+aiosqlite,0
+algoliasearch,~=2.5
+anyio,>=3.4.0
+aredis,0
+asgiref,~=3.0
+astunparse,0
+async_generator,~=1.10
+asyncpg,~=0.22.0
+asynctest,==0.13.0
+attrs,>=20
+austin-python,~=1.0
+blinker,0
+boto3,0
+botocore,~=1.13
+bottle,>=0.12
+bytecode,0
+cassandra-driver,~=3.24.0
+cattrs,<23.1.1
+celery,~=4.4
+cfn-lint,~=0.53.1
+channels,~=3.0
+cherrypy,>=17
+click,==7.1.2
+cohere,0
+confluent-kafka,~=1.9.2
+coverage,0
+cryptography,<39
+daphne,0
+databases,0
+datadog-lambda,>=4.66.0
+ddsketch,>=3.0.0
+django,>=2.2
+django-pylibmc,>=0.6
+django-q,0
+django-redis,>=4.5
+django_hosts,~=4.0
+djangorestframework,>=3.11
+docker,0
+dogpile.cache,~=0.9
+dramatiq,0
+elasticsearch,~=7.13.0
+elasticsearch1,~=1.10.0
+elasticsearch2,~=2.5.0
+elasticsearch5,~=5.5.0
+elasticsearch6,~=6.8.0
+elasticsearch7,~=7.13.0
+elasticsearch7[async],0
+elasticsearch8,~=8.0.1
+elasticsearch[async],0
+envier,==0.5.1
+exceptiongroup,0
+falcon,~=3.0
+fastapi,~=0.64.0
+flask,~=0.12.0
+flask-caching,~=1.10.0
+flask-login,~=0.6.2
+gevent,~=20.12.0
+git+https://github.com/gnufede/pytest-memray.git@24a3c0735db99eedf57fb36c573680f9bab7cd73,0

```

Comment:
we probably don't want this, right?

seems like we are also getting test dependencies in here, which we might not want because what we require for our tests might not indicate what we are compatible with.

Reply:
Looking through the riotfile, I'm not even sure that I could accurately distinguish "test requirements" from "what the library is compatible with". Part of the thinking behind this approach is that what our tests install is the definition of what the library is compatible with.